lib/rucio/db/sqla/models.py

```diff
@@ -1251,6 +1252,37 @@ class LifetimeExceptions(BASE, ModelBase):
                    ForeignKeyConstraint(['account'], ['accounts.account'], name='LIFETIME_EXCEPT_ACCOUNT_FK'))
 
 
+class DidsFollowed(BASE, ModelBase):
+    """Represents the datasets followed by an user"""
+    __tablename__ = 'dids_followed'
+    scope = Column(String(SCOPE_LENGTH))
+    name = Column(String(NAME_LENGTH))
+    account = Column(String(25))
+    did_type = Column(DIDType.db_type(name='DIDS_FOLLOWED_TYPE_CHK'))
+    expires_at = Column(DateTime)
+    _table_args = (PrimaryKeyConstraint('scope', 'name', 'account', name='DIDS_FOLLOWED_PK'),
+                   CheckConstraint('SCOPE IS NOT NULL', name='DIDS_FOLLOWED_SCOPE_NN'),
+                   CheckConstraint('NAME IS NOT NULL', name='DIDS_FOLLOWED_NAME_NN'),
+                   CheckConstraint('DID_TYPE IS NOT NULL', name='DIDS_FOLLOWED_DID_TYPE_NN'),
+                   ForeignKeyConstraint(['account'], ['accounts.account'], name='DIDS_FOLLOWED_ACCOUNT_FK'))
+
+
+class FollowEvents(BASE, ModelBase):
+    """Represents the events affecting the datasets which are followed"""
+    __tablename__ = 'follow_events'
+    scope = Column(String(SCOPE_LENGTH))
+    name = Column(String(NAME_LENGTH))
+    account = Column(String(25))
+    did_type = Column(DIDType.db_type(name='FOLLOW_EVENTS_TYPE_CHK'))
+    event_type = Column(String(1024))
+    payload = Column(LargeBinary(255))
```

Comment:
is the payload definition and size correct here?

Reply:
Martin Sir had asked to keep the payload as a blob so that's why LargeBinary. And for the size I checked similar definitions in the models.py file and they had used 255 for LargeBinary. So I kept it the same.. Should it be increased?

Reply:
For other tables we use `Column(Text)` for this. I would just do this here as well.

Reply:
Okay Sir. I will change it to `Column(Text)`